src/Domain/DonationService.php

```diff
@@ -345,7 +336,7 @@ function () use ($donation) {
             );
         }
 
-        if ($donation->getPsp() === 'stripe') {
+        if ($donation->getPsp() === 'stripe' && $donation->getDonationStatus() === DonationStatus::Pending) {
```

Comment:
```suggestion
        // Regular Giving enrolls donations with `DonationStatus::PreAuthorized`, which get Payment Intents later instead.
        if ($donation->getPsp() === 'stripe' && $donation->getDonationStatus() === DonationStatus::Pending) {
```
Maybe just because of the context shown by default in GitHub but it took me a minute to confirm the rationale for this. Maybe worth a comment for future glances at this area?

Reply:
Thanks! Suggestion applied.